kong/llm/drivers/shared.lua

```diff
@@ -504,6 +506,9 @@ function _M.pre_request(conf, request_table)
     kong.ctx.shared.ai_prompt_tokens = (kong.ctx.shared.ai_prompt_tokens or 0) + prompt_tokens
   end
 
+  local start_time_key = "ai_request_start_time_" .. string.sub(conf.__plugin_id, -12)
+  kong.ctx.shared[start_time_key] = ngx.now()
```

Comment:
seems we can just use `kong.ctx.plugin`?

Reply:
Good catch I am also going to use the pluginName as key instade of the pluginId as only one plugin is possible by request. It will avoid the usage of stringSub and also facilitate the test